mattermost-plugin/webapp/src/plugin.scss

```diff
@@ -5,6 +5,24 @@
     align-items: center;
 }
 
-.focalboard-body .feature-global-header > header {
+.focalboard-body .feature-global-header>header {
     z-index: 1000;
-}
\ No newline at end of file
+}
+
+.feature-global-header {
```

Comment:
This is going to affect the `.feature-global-header` of outside focalboard too. if you combine this with the previous one you can make it better. Something like:

```css
.focalboard-body .feature-global-header {
    >header {
        z-index: 1000;
    }
    .Menu {
...
```

Reply:
@jespino Done.